manifests/init.pp

```diff
@@ -107,5 +123,67 @@
     ],
     require     => Package['chocolatey.server'],
   }
-  # technically you may only need IIS_IUSRS but I have not tested this yet.
+
+  # configure chocolatey server settings
+  file { "${_chocolatey_server_location}/web.config":
+    ensure  => file,
+    content => epp('chocolatey_server/web.config.epp', {
+      'allowOverrideExistingPackageOnPush' => $allow_package_override,
+      'apiKey'                             => $apikey,
+      'requireApiKey'                      => $require_apikey,
+    }),
```

Comment:
Sure, I can try to rework it into an ERB template. I'll let you know when complete.